src/app/app.routing.ts

```diff
@@ -7,6 +7,11 @@ import { LoginComponent } from './authentication/login.component';
 import { ForgotPasswordComponent } from './authentication/forgot-password.component';
 import { ResetPasswordComponent } from './authentication/reset-password.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
+import { SplashscreenComponent } from './splashscreen/splashscreen.component';
```

Comment:
in other words, these are not needed in this PR.

ps: have fun resolving merge conflicts.